### ISSUE TITLE
Unable to play Seamless Launch on an agenda installed before starting turn

### ISSUE THREAD

**Issue by OtisF** [mtgred/netrunner#6244]
Link to bug replay: https://jinteki.net/bug-report/f7344cd5-8bd0-4748-9243-978a6edbb7ec?b=0

Description:

On my last turn in the game, before clicking the "Start turn" button, I rezzed a Political Dealings and a Spin Doctor, and used PD to install an agenda I drew. I attempted to play a Seamless Launch on that agenda but the UI would not accept me clicking on it. 

My understanding was that before clicking "Start Turn" it is essentially considered the runners turn, hence why I can have a Daily Business Show filter from the Spin Doctor rez whilst also getting it on my mandatory draw afterwards. Therefore the agenda was not installed during my turn, and so Seamless Launch should be playable.

**Comment by lostgeek** [comment]
Yeah, you should have been able to use Seamless Launch there. This must be a bug.

**Comment by francescopellegrini** [comment]
Duplicate of #5888

**Comment by francescopellegrini** [comment]
Might as well close this.
